### PR TITLE
Added additional RCA for cluster deployment timeout

### PR DESCRIPTION
**Pull request by mandre**
Example job: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/logs/release-openshift-ocp-installer-e2e-openstack-4.4/362

### CODE REVIEW

**Comment by mandre** [comment]
Hold on, I noticed another cluster deployment timeout with a similar error.

**Comment by mandre** [comment]
We could consider merging both rules into one, however I'm not sure that we can assume the "failed to initialize the cluster" error is equivalent to a timeout while deploying.

**Review by pierreprinetti — APPROVED** [review]

File: pkg/rca/rca.go [hunk 1]
```diff
@@ -22,6 +22,21 @@ var (
 			CauseClusterTimeout,
 		),
 
+		infraFailureIfMatchBuildLogs(
+			"failed to initialize the cluster: Cluster operator [\\w-]+ is still updating",
```
Comment (pierreprinetti) [inline pkg/rca/rca.go:26]: you can ```s|"|`|g``` this line if you want to avoid escaping.

**Comment by pierreprinetti** [comment]
Do we still have to hold on? Feel free to merge if it's ready

**Pull request merged**